cernopendata/uwsgi.ini

```diff
@@ -0,0 +1,7 @@
+[uwsgi]
+http = 0.0.0.0:5000
+module = cernopendata.wsgi:application
+master = true
+die-on-term = true
+processes = 2
```

Comment:
Make configurable see (here)[https://github.com/unbit/uwsgi/issues/1604#issuecomment-321320233]

Reply:
👍 for configurability... and instead of using a file, perhaps just make a long CLI call?  

`uwsgi --http-socket $ENVVAR1 --processes $ENVVAR2 ...`